podcasts.xcodeproj/project.pbxproj

```diff
@@ -12994,14 +12990,12 @@
 					"@executable_path/Frameworks",
 				);
 				LOCALIZATION_PREFERS_STRING_CATALOGS = YES;
-				MARKETING_VERSION = 1.0;
 				MTL_ENABLE_DEBUG_INFO = INCLUDE_SOURCE;
 				MTL_FAST_MATH = YES;
 				PRODUCT_BUNDLE_IDENTIFIER = au.com.shiftyjelly.podcasts.Clip;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = "";
-				"PROVISIONING_PROFILE_SPECIFIER[sdk=iphoneos*]" = "Pocket Casts App Clip Development";
-				SKIP_INSTALL = NO;
+				PROVISIONING_PROFILE_SPECIFIER = "Pocket Casts App Clip Development";
+				SKIP_INSTALL = YES;
```

Comment:
With `SKIP_INSTALL` set to `NO` for the App Clip target, Xcode is able to archive and export the app, but `xcodebuild` via Fastlane fails with:

```
error: exportArchive exportOptionsPlist error for key "method" expected one {} but found app-store
```

I don't know if this change will have downstream effects in the distribution capability for the App Clip.

For reference, from https://developer.apple.com/documentation/xcode/build-settings-reference

> SKIP_INSTALL: If enabled, don’t install built products even if deployment locations are active.

Reply:
Nah I'm pretty sure `SKIP_INSTALL=NO` is for things like when you build a macOS (GUI or CommandLine) app and want the built app to be installed in `/Applications` / `/usr/local/bin` or something like that? So shouldn't have any incidence in this case 👍